code/modules/uplink/uplink_items/stealthy_tools.dm

```diff
@@ -18,6 +18,14 @@
 	item = /obj/item/card/id/advanced/chameleon
 	cost = 2
 
+/datum/uplink_item/stealthy_tools/agent_card_advanced
+	name = "Advanced Agent Identification Card"
+	desc = "Created by Cybersun Industries to be the ultimate for field operations, this upgraded Agent ID \
+	comes with all the fluff of the original, but with an upgraded microchip - allowing for the storage of all \
+	standard Nanotrasen access codes in one conveinent package. Now in glossy olive by default!"
```

Comment:
The ingame sprite seems to be black rather than olive, so "glossy black" or another descriptor would be better:
![image](https://user-images.githubusercontent.com/57483089/169547167-8265ebdd-67aa-4b7d-af04-664b583ffb60.png)
```suggestion
	standard Nanotrasen access codes in one conveinent package. Now in glossy black by default!"
```

Reply:
~~This is more matte than glossy~~